app/celery/provider_tasks.py

```diff
@@ -92,7 +92,7 @@ def check_sms_delivery_receipt(self, message_id, notification_id, sent_at):
 
 
 @notify_celery.task(
-    bind=True, name="deliver_sms", max_retries=48, default_retry_delay=300
+    bind=True, name="deliver_sms", max_retries=12, default_retry_delay=300
```

Comment:
Just double checking on the change of `48` to `12` here for both this and `check_sms_delivery_receipt` above; apologies if we discussed this and I forgot!

Reply:
No, I got inspired and made this change on my own because I was trying to reduce the number of times we pulled down a job from s3 to get a phone number.  But I'd be better off increase the time on the job cache, will change.